### PR TITLE
Move getting started and refactor build system

### PR DESCRIPTION
**Pull request by laszlof**
Dont merge yet.

### CODE REVIEW

File: getting-started/git.md [hunk 1]
```diff
@@ -1,87 +0,0 @@
----
```
Comment (adrian-enspired) [inline getting-started/git.md:1]: just out of curiosity, we don't want this article?

**Review by adrian-enspired — APPROVED** [review]

File: getting-started/git.md (hunk 1 above)
Comment (laszlof) [reply getting-started/git.md:1]: No, its going to get refactored and probably moved.

File: bin/build [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      title: contents.meta.title,
+      subject: contents.meta.subject,
+      route: getRouteFromFilename(file_name)
+    }];
+    return article;
+  }
+  for (var file of fs.readdirSync(link)) {
+    var file_name = link + '/' + file;
+    if (
+      excluded_files.indexOf(file) > -1 ||
```
Comment (casmit) [inline bin/build]: Can use ```excluded_files.includes(file)```.

File: bin/build [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    title: cat.title,
+    articles: getArticlesFromPath(cat.link)
+  });
+}
+
+json.writeFileSync('./build/data.json', output, {spaces: 2});
+
+function getArticlesFromPath(link) {
+  var articles = {};
+  if (! fs.existsSync(link)) {
```
Comment (casmit) [inline bin/build:39]: You use ```existsSync``` and ```lstatSync().isDirectory()``` on 58. Should probably choose one or the other for consistency.

File: bin/build [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    if (! articles.hasOwnProperty(contents.meta.subject)) {
+      articles[contents.meta.subject] = [];
+    }
+    articles[contents.meta.subject].push(article);
+  }
+
+  return articles;
+};
+
+function getRouteFromFilename(file_name) {
```
Comment (casmit) [inline bin/build:80]: Can use ```path.dirname()```, don't need new function.

File: bin/build [hunk 5]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+var md = require('meta-marked'),
```
Comment (casmit) [inline bin/build]: ```var``` for each one of these, ```const``` if you want to be cool.

**Review by casmit — CHANGES_REQUESTED** [review]

File: bin/build (hunk 3 above)
Comment (laszlof) [reply bin/build:39]: `lstatSync` throws an exception if the file doesnt exist. They serve different purposes.

File: bin/build (hunk 4 above)
Comment (laszlof) [reply bin/build:80]: This isnt returning the dirname, its returning the file path without the extention.

**Pull request merged**